assets/eip-5725/contracts/IVestingNFT.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.0;
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+
+/**
+ * @title Non-Fungible Vesting Token Standard
+ * @notice A non-fungible token standard used to vest tokens (ERC-20 or otherwise) over a vesting release curve
+ *  scheduled using timestamps.
+ * @dev Because this standard relies on timestamps for the vesting schedule, it's important to keep track of the
+ *  tokens claimed per Vesting NFT so that a user cannot withdraw more tokens than alloted for a specific Vesting NFT.
+ */
+interface IVestingNFT is IERC721 {
+    event PayoutClaimed(uint256 indexed tokenId, address indexed recipient, uint256 _claimAmount);
+
+    /**
+     * @notice Claim the pending payout for the NFT
+     * @dev MUST grant the claimablePayout value at the time of claim being called
+     * MUST revert if not called by the token owner or approved users
+     * SHOULD revert if there is nothing to claim
+     * @param tokenId The NFT token id
+     * @return amountClaimed The amount of tokens claimed in this call
+     */
+    function claim(uint256 tokenId) external returns (uint256 amountClaimed);
```

Comment:
I don't believe it's standard to return the `amountClaimed` here. 

I'm recommending that this definition be changed to: `function claim(uint256 tokenId) external`